File: php/SendEmailExample.php
<?php
require_once "MailproClient.php";

$client = new MailproClient("YOUR_API_KEY");

try {
    $response = $client->sendEmail(
        "you@example.com",
        "recipient@example.com",
        "Hello from Mailpro PHP!",
        "<p>This is a test email sent via the Mailpro API using PHP.</p>"
    );
    print_r($response);
} catch (Exception $e) {
    echo "Error: " . $e->getMessage();
}
?>

File: java/SendEmailExample.java
public class SendEmailExample {
    public static void main(String[] args) {
        try {
            MailproClient client = new MailproClient("YOUR_API_KEY");

            String subject = "Test Email from Java";
            String from = "sender@example.com";
            String to = "recipient@example.com";
            String html = "<h1>Hello from Java!</h1><p>This is a test email.</p>";

            String response = client.sendEmail(subject, from, to, html);
            System.out.println("Send Email Response: " + response);

        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
